provision/acc_provision/apic_provision.py

```diff
@@ -1431,6 +1431,37 @@ def cloudfoundry_tn(self):
                    ["app_profile"])
         app_epg_name = (
             self.config["cf_config"]["default_endpoint_group"]["group"])
+        custom_epg_name = (
+            self.config["aci_config"]["cloudfoundry"]["custom_epg"])
+        isolated_segment = (
+            self.config["aci_config"]["cloudfoundry"]["isolated_segment"])
+        children_epg_objs, children_apic_objects = [], []
+
+        if isolated_segment:
```

Comment:
Looks like you are assuming a single isolation segment, but we need to support multiple.